health/service_test.go

```diff
@@ -42,18 +45,18 @@ func TestHealthCheck(t *testing.T) {
 	rr := httptest.NewRecorder()
 	// Create a request to pass to our handler. We don't have any query parameters for now, so we'll
 	// pass 'nil' as the third parameter.
-	req := http.Request{Method: "GET", RequestURI: "/health"}
+	req, err := http.NewRequest("GET", "/health", bytes.NewBuffer(nil))
```

Comment:
You can just pass nil as third arg here, no need to create a new buffer that never get's used.

Reply:
Nice. I figured that would error down the line somewhere but it looks like that totally works.